src/logic/small.lean

```diff
@@ -118,6 +119,34 @@ instance small_sum {α β} [small.{w} α] [small.{w} β] : small.{w} (α ⊕ β)
 instance small_set {α} [small.{w} α] : small.{w} (set α) :=
 ⟨⟨set (shrink α), ⟨equiv.set.congr (equiv_shrink α)⟩⟩⟩
 
+theorem small_range {ι : Type u} {α} (f : ι → α) : small.{u} (set.range f) :=
+begin
+  let S := setoid.ker f,
```

Comment:
Does this generalize to some `small_of_surjective`?

Reply:
Yes, see my commit. I also golfed the proof.

Reply:
Thanks! You made it look so simple, haha